### PR TITLE
ocrvs:3421: Fixed multiple time currency appearing issue

### PR DESCRIPTION
**Pull request by soumita-dsi**

### CODE REVIEW

File: packages/client/src/views/CorrectionForm/CorrectionSummary.tsx [hunk 1]
```diff
@@ -145,9 +145,15 @@ class CorrectionSummaryComponent extends React.Component<IFullProps, IState> {
         this.props.offlineResources.config.CURRENCY &&
         this.props.offlineResources.config.CURRENCY['isoCode']
     })[0].symbol
-    ;(
-      this.group.fields[0].nestedFields as any
-    ).REQUIRED[0].label.defaultMessage += ` ${currency}`
+
+    if (
+      (this.group.fields[0].nestedFields as any).REQUIRED[0].label
+        .defaultMessage === 'Total'
```
Comment (atiqzaman-dsi) [inline packages/client/src/views/CorrectionForm/CorrectionSummary.tsx:151]: I think we need french translation instead of hard coded value

**Review by atiqzaman-dsi — CHANGES_REQUESTED** [review]

File: packages/client/src/views/CorrectionForm/CorrectionSummary.tsx (hunk 1 above)
Comment (soumita-dsi) [reply packages/client/src/views/CorrectionForm/CorrectionSummary.tsx]: Updated the code. Farajaland PR link:

File: packages/client/src/views/CorrectionForm/CorrectionSummary.tsx [hunk 2]
```diff
@@ -145,9 +145,13 @@ class CorrectionSummaryComponent extends React.Component<IFullProps, IState> {
...
         this.props.offlineResources.config.CURRENCY['isoCode']
     })[0].symbol
+
     ;(
       this.group.fields[0].nestedFields as any
-    ).REQUIRED[0].label.defaultMessage += ` ${currency}`
+    ).REQUIRED[0].label.defaultMessage = this.props.intl.formatMessage(
+      messages.correctionSummaryTotalPaymentLabel,
+      { currency }
+    )
```
Comment (rikukissa) [inline packages/client/src/views/CorrectionForm/CorrectionSummary.tsx:154]: Where is this label being used? Could the internationalisation bit be moved into the rendering logic? Mutating the form field object in the `componentDidMount` method to make the currency label render correctly seems a bit like a code smell.

No need to change anything now as this component seems massive and very complex. More of a question for the future I guess :)

Comment (soumita-dsi) [reply packages/client/src/views/CorrectionForm/CorrectionSummary.tsx:154]: > Where is this label being used? Could the internationalisation bit be moved into the rendering logic? Mutating the form field object in the `componentDidMount` method to make the currency label render correctly seems a bit like a code smell.
> 
> No need to change anything now as this component seems massive and very complex. More of a question for the future I guess :)

This label is being used in fees collection part of correction summary page, is defined in payment.ts file` nestedFields: {
        REQUIRED: [
          {
            name: 'totalFees',
            type: 'NUMBER',
            label: {
              defaultMessage: '',
              id: 'form.field.label.totalFees',
              description: 'Label for input Reason for Change'
            },
            required: true,
            initialValue: '',
            validate: [],
            mapping: {}
          },
          {
            name: 'proofOfPayment',
            type: 'SIMPLE_DOCUMENT_UPLOADER',
            label: messages.correctionSummaryProofOfPayment,
            description: messages.correctionSummaryProofOfPayment,
            required: true,
            initialValue: '',
            validate: [],
            mapping: {}
          }
        ]
      }`

**Review by atiqzaman-dsi — APPROVED** [review]

**Pull request merged**